Src/BootCamp.Chapter/Checks.cs

```diff
@@ -17,25 +17,26 @@ public static class Checks
         public static int PromptInt(string message)
         {
             // To do: call your implementation. 
```

Comment:
Please remove the comment now, as it is no longer needed- you understood what you have to do